resources/definition.rb

```diff
@@ -0,0 +1,40 @@
+property :definition, String,
+         name_property: true,
+         description: 'Version of Ruby to install'
+
+property :prefix_path, String,
+         default: lazy { '/usr/local/ruby' },
```

Comment:
Is there a reason to lazy eval this string?

Reply:
Not any more... oops thanks :)